mesheryctl/internal/cli/root/system/error.go

```diff
@@ -83,3 +84,7 @@ func ErrProcessingMctlConfig(err error) error {
 func ErrRestartMeshery(err error) error {
 	return errors.New(ErrRestartMesheryCode, errors.Alert, []string{"Error restarting Meshery"}, []string{err.Error()}, []string{"Meshery is not running"}, []string{"Restart Meshery instance"})
 }
+
+func ErrK8SQuery(err error) error {
+	return errors.New(ErrK8sQueryCode, errors.Alert, []string{err.Error()}, []string{"!! cannot query the Kubernetes API. See https://docs.meshery.io/reference/error-codes"}, []string{"Meshery is not running"}, []string{"Restart Meshery instance"})
```

Comment:
@piyushsingariya I'm confused. If the Kubernetes API isn't available, why is the probably cause suggested to be that Meshery Server isn't running? Why is the probable cause not Kubernetes is not running or is inaccessible and that the suggested remediation is to verify connectivity between Meshery Server and Kubernetes and to verify that kubeconfig certificates have permission to query Kubernetes API?

Reply:
The line is soooooooooooooooooooooo long, I missed to remove those. 😂

Reply:
corrected!